docs/history/changelog.md

```diff
@@ -36,6 +36,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
 
 * [#1086](https://github.com/open-contracting/standard/pull/1086) Recommend the use of strings rather than integers for identifiers.
 * [#1094](https://github.com/open-contracting/standard/pull/1094) Clarify the uniqueness of `Organization.id`
+* [#1112](https://github.com/open-contracting/standard/pull/1112) Strengthen some normative statements in schema
```

Comment:
Can you explicitly mention the three changes, so a reader of the changelog doesn't need to open the PR/issue?

Reply:
Done